cmd/client-keystone-auth/main.go

```diff
@@ -70,59 +70,92 @@ func promptForString(field string, r io.Reader, show bool) (result string, err e
 			return "", fmt.Errorf("error reading input for %s", field)
 		}
 	}
+
+	// intentionally ignore "unexpected newline" errors to allow read empty lines from the console
+	if err != nil && err.Error() == "unexpected newline" {
```

Comment:
This is sort of hack, but I didn't find a better solution to allow reading empty lines from the console